gym/lib/gym/generators/package_command_generator_legacy.rb

```diff
@@ -40,6 +42,15 @@ def pipe
         [""]
       end
 
+      # Place where the IPA file will be created, so it can be safely moved to the destination folder
+      def temporary_output_path
+        unless Gym.cache[:temporary_output_path]
+          Gym.cache[:temporary_output_path] = "#{Tempfile.new('gym').path}.gym_output"
```

Comment:
I've copied code from https://github.com/fastlane/fastlane/blob/master/gym/lib/gym/generators/package_command_generator_xcode7.rb#L40. So the two package generators have the same implementation.

Should I change it? If yes, should I change it only in this file or also in the `package_command_generator_xcode7.rb`?